Resource description given to LSF fails in Python driver

**Describe the bug**
```
E           RuntimeError: Command "/global/bin/bsub -R 'rusage=[mem=20MB]' -J dummy 
/private/f_scout_ci/pytest-tmp/tmp.zneeX4MS45/test_lsf_driver_masks_returnco4/.lsf_submit_b75fhab8.sh /private/f_scout_ci/pytest-tmp/tmp.zneeX4MS45/test_lsf_driver_masks_returnco4"
failed after 10 retries with error Error in select section: Error near "rusage" : incorrect section name, 
resource name, or resource function name. Job not submitted.
```
**To reproduce**
Steps to reproduce the behaviour:
Run LSF integration tests.

**Expected behaviour**
No error


**Environment**
 - OS: [e.g. RHEL7]
 - ERT/Komodo release: [e.g. 2.18 or 2020.12.00]
 - Python version
 - Remote/HPC execution involved: [yes|no]

**Additional context**
Add any other context about the problem here.

--- a/tests/integration_tests/scheduler/test_lsf_driver.py
+++ b/tests/integration_tests/scheduler/test_lsf_driver.py
@@ -121,7 +121,7 @@ async def test_lsf_driver_masks_returncode(
 
 
 async def test_submit_with_resource_requirement(tmp_path):
-    resource_requirement = "rusage=[mem=20MB]"
+    resource_requirement = "select[cs && x86_64Linux]"
     driver = LsfDriver(resource_requirement=resource_requirement)
     await driver.submit(0, "sh", "-c", f"echo test>{tmp_path}/test")
     await poll(driver, {0})

--- a/tests/unit_tests/scheduler/test_lsf_driver.py
+++ b/tests/unit_tests/scheduler/test_lsf_driver.py
@@ -135,9 +135,9 @@ async def test_submit_with_default_queue():
 
 @pytest.mark.usefixtures("capturing_bsub")
 async def test_submit_with_resource_requirement():
-    driver = LsfDriver(resource_requirement="rusage[mem=512MB:swp=1GB]")
+    driver = LsfDriver(resource_requirement="select[cs && x86_64Linux]")
     await driver.submit(0, "sleep")
-    assert "-R rusage[mem=512MB:swp=1GB]" in Path("captured_bsub_args").read_text(
+    assert "-R select[cs && x86_64Linux]" in Path("captured_bsub_args").read_text(
         encoding="utf-8"
     )
 

--- a/tests/unit_tests/scheduler/test_scheduler.py
+++ b/tests/unit_tests/scheduler/test_scheduler.py
@@ -520,7 +520,7 @@ def test_scheduler_create_lsf_driver():
     bkill_cmd = "foo_bkill_cmd"
     bjobs_cmd = "bar_bjobs_cmd"
     bhist_cmd = "com_bjobs_cmd"
-    lsf_resource = "rusage[mem=512MB:swp=1GB]"
+    lsf_resource = "select[cs && x86_64Linux]"
     queue_config_dict = {
         "QUEUE_SYSTEM": "LSF",
         "QUEUE_OPTION": [

